.gitignore

```diff
@@ -2,4 +2,8 @@
 mysql_config.json
 __pycache__
 .DS_Store
-.pre-commit-config.yaml
\ No newline at end of file
+.pre-commit-config.yaml
+dumps
```

Comment:
Please do not add these files to .gitignore. it is just your local thing and should not be in the project